public/js/migrate-app.js

```diff
@@ -68885,11 +68885,9 @@ Tree.prototype.render = function (root, entries, onclick) {
 
   function backRow () {
     if (root === '/' || root === '' || root === '.') return
-    return yo`<li class='entry-back' onclick=${backButton}>
-      <a href='javascript:void(0)'>
-        <span class="name">..</span>
-      </a>
-    </li>`
+    return yo`<tr class='entry-back' onclick=${backButton} href='javascript:void(0)'>
```

Comment:
i think we don't have to worry about migrate-app.js anymore, it was the old app and we're going to toss it